src/lib/config.ts

```diff
@@ -339,6 +339,11 @@ export const DEFAULT_SUPERVISOR_POLL_INTERVAL = intVar(
 	10 * MINUTES,
 );
 
+export const DEFAULT_SUPERVISOR_REPORT_INTERVAL = intVar(
```

Comment:
Having this as an injected env var doesn't offer much, unless you also plan to enforce this value in all state-GETs like we do for the min poll interval.
See: https://github.com/balena-io/open-balena-api/blob/fd8899bf269c998cb182bfcc8d5600b1a5ea1780/src/features/device-state/state-get-utils.ts#L6